geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/kafka/commands/CreateCommand.scala

```diff
@@ -0,0 +1,60 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.kafka.commands
+
+import com.beust.jcommander.{JCommander, Parameters}
+import com.typesafe.scalalogging.LazyLogging
+import org.locationtech.geomesa.kafka.KafkaDataStoreHelper
+import org.locationtech.geomesa.tools.common.{CLArgResolver, FeatureTypeNameParam, FeatureTypeSpecParam}
+import org.locationtech.geomesa.tools.kafka.ProducerKDSConnectionParams
+import org.locationtech.geomesa.tools.kafka.commands.CreateCommand.CreateParameters
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+
+class CreateCommand(parent: JCommander) extends CommandWithKDS(parent) with LazyLogging {
+  override val command = "create"
+  override val params = new CreateParameters()
+
+  override def execute() = {
+    val sft = CLArgResolver.getSft(params.spec, params.featureName)
+    val featureName = params.featureName
+
+    val sftString = SimpleFeatureTypes.encodeType(sft)
+    logger.info(s"Creating '$featureName' using a KafkaDataStore with spec " +
+      s"'$sftString'. Just a few moments...")
+
+    val streamingSFT = KafkaDataStoreHelper.createStreamingSFT(sft, zkPath)
+
+    try {
+      logger.info("Creating GeoMesa Kafka schema...")
+      ds.createSchema(streamingSFT)
+
+      if (ds.getSchema(featureName) != null) {
+        logger.info(s"Feature '$featureName' with spec " +
+          s"'$sftString' successfully created in Kafka with zkPath '$zkPath'.")
+        println(s"Created feature $featureName")
+      } else {
+        logger.error(s"There was an error creating feature '$featureName' with spec " +
+          s"'$sftString'. Please check that all arguments are correct " +
+          "in the previous command.")
+      }
+    } catch {
+      // if the schema already exists at the specified zkPath
+      case e: IllegalArgumentException => {
+        logger.warn(e.getMessage)
```

Comment:
Does the message provide enough info back to the user? or should we explicitly say what's in your comment there.